### DISCUSSION TITLE
Fine tune the autocomplete algorithm when typing mentions

### DISCUSSION THREAD

**Discussion by martinbonnin** [community/community#6239] [Other Feature Feedback, Questions, & Ideas]
Hi! 

What is the current sort algorithm when typing `@` mentions in an issue comment? It happens very often that the wrong person gets tagged because it sorts first even if someone actively participating the conversation and having the same prefix would be a better match. 

This is annoying as it creates noise for someone not directly subscribed to the issue. Is this something that can be improved?

**Comment by huehnerlady** [comment]
I agree, on top of that it is odd that you get suggestions from all over github instead getting the members in your organization first
